src/components/StatsBoxGrid.tsx

```diff
@@ -233,18 +127,90 @@ const GridItem: React.FC<IGridItemProps> = ({ metric, dir }) => {
   )
 
   return (
-    <Box>
+    <Box
```

Comment:
This should be a flex component. I would suggest `VStack` with `spacing={0}` and remove the `display="flex"` and `flexDirection="column"`

```tsx
<VStack
  spacing={0}
  // The other props
/>
```